src/components/login-form.tsx

```diff
@@ -26,48 +28,138 @@ export type LoginFormProps = {
   onSubmit?: SubmitHandler<FormType>;
 };
 
+type InputConstantsProps = {
+  control: any;
+  className: string;
+  labelClassname: string;
+  showError: boolean;
+};
+
+type InputVariableProps = {
+  name: 'name' | 'email' | 'password' | 'confirmPassword';
+  label: string;
+  placeholder: string;
+  secureTextEntry?: boolean;
+};
+
+type InputProps = {
```

Comment:
Consider moving all types to their own `types.ts` file, which will reduce the amount of code in the component, and also will improve the readability.